test/pseudo-tty/test-async-wrap-getasyncid-tty.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// see also test/sequential/test-async-wrap-getasyncid.js
```

Comment:
This could get outdated easily and I doubt that we actually need that comment. The same applies to the other file accordingly.

Reply:
imo, better to have something than nothing